### DISCUSSION TITLE
Where does this request come from?

### DISCUSSION THREAD

**Discussion by devfle** [WordPress/gutenberg#41530] [Q&A] [answered]
I do not want my client to be able to change the General Settings in the Dashboard. For this reason, I have removed the capability: Manage_Options. (https://wordpress.org/support/article/roles-and-capabilities/#manage_options)

In Gutenberg itself, a request is made at some point to the following rest endpoint:

`wp-json/wp/v2/settings?_locale=user`

This request fails due to the missing capability. Unfortunately I could not find out where this request comes from and how important it is. Maybe someone can help me?

**Comment by talldan** [comment]
@devfle What are you doing when the request is made? Is this in one of the site/post/widgets editor or all?

**Comment by devfle** [comment, marked as answer]
Sorry for the late feedback. The issue has been resolved with the current version of the Gutenberg editor.
